pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -7,18 +7,57 @@
 # include <string.h>
 # include <stdarg.h>
 # include <stdlib.h>
+# include <stdio.h>
 
 /*

```

Comment:
hmm, this comment should be above `array_create`

Reply:
I am not sure it should stay, but if it does, then yes, I will put it above `array_create`.

Reply:
Am guessing you are talking about the comment `allocation`, I shall put it above `array_create`.